diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -157,13 +156,20 @@ controller_interface::return_type DiffDriveController::update(
     return controller_interface::return_type::ERROR;
   }
 
-  const auto age_of_last_command = current_time - last_command_msg->header.stamp;
-  // Brake if cmd_vel has timeout, override the stored command
-  if (age_of_last_command > cmd_vel_timeout_)
-  {
-    last_command_msg->twist.linear.x = 0.0;
-    last_command_msg->twist.angular.z = 0.0;
-  }
+//   if (last_command_msg->header.stamp.sec != 0 and last_command_msg->header.stamp.nanosec != 0)
```

Comment:
I think we discussed this enough but basically there should be a common policy on this and we should offer only one solution. An optional addition can be to opt-out of this feature altogether, in that case one could publish commands with timestamps from different time sources, we wouldn't know.